schema/sql/settings.sql

```diff
@@ -0,0 +1,10 @@
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SET check_function_bodies = false;
+SET xmloption = content;
+SET client_min_messages = warning;
+SET row_security = off;
```

Comment:
Why would you set row security off?

Reply:
You tell me. That is how you setup production.